README.md

```diff
@@ -137,6 +138,36 @@ The available flags are:
   - `add-module-exports` toggles whether the UMD output of ez-build should be backwards compatible with AMD and CJS module formats. If this flag is specified, ez-build will ensure any module with a single `export default` will not export an object with a `default` key. This flag is disabled by default. It is only recommended you use this flag if you *must* keep backwards compatibility with legacy code.
   - `es2017` toggles support for compiling code ES2017 code. When ES2017 is ratified this flag will be removed and the ES2017 preset will be added by default. This flag is diabled by default.
 
+### `@<path>`
+
+Reads ez-build options from the file at `<path>`, resolved from the current working directory. This can be used to share configuration among different builds. For instance, a common scenario is to share the same build configuration between `--production` and `--interactive` builds in npm scripts, like so:
+
+```json
+{
+  "build": "ez-build --production --include \"js:**/*.js,js:**/*.jsx\" --flags es2017",
```

Comment:
if this supports glob patterns, you could write it as `js:**/*.{js,jsx}` - but it may also potentially benefit from not coupling to a structure that has a `js` folder - it could well be `scripts` or `src` or whatever.